extensions/cms/package.json

```diff
@@ -544,15 +544,6 @@
 					"dark": "resources/dark/refresh_inverse.svg"
 				}
 			},
-			{
-				"command": "cms.resource.refreshCmsServer",
```

Comment:
so once we remove this he original "refresh" takes care of refreshing cms automatically?